### ISSUE TITLE
Ignite cache cannot expire

### ISSUE THREAD

**Issue by zhoudingyun** [vert-x3/vertx-ignite#95]
**Questions**
Set the cache and set the expiration time. When the expiration time is exceeded, the cache expires.
However, set the cache and set the expiration time. If you get the cache, the cache will not expire. This is a very serious problem. I found that the expiration policy was cleared when getting the cache.
![image](https://user-images.githubusercontent.com/32148584/98216735-6aa0cf80-1f84-11eb-86de-f6d52eafd423.png)

**Version**
vertx-ignite 3.9.0 or laster.

**Comment by zyclonite** [comment]
this is a workaround https://github.com/vert-x3/vertx-ignite/issues/63 of an ignite issue [IGNITE-7401](https://issues.apache.org/jira/browse/IGNITE-7401)

**Comment by zhoudingyun** [comment]
@zyclonite The problem I encountered is different from this one.

**Comment by zyclonite** [comment]
can you provide a reproducer for your use-case?

**Comment by zhoudingyun** [comment]
@zyclonite 
case one:
Set up the cache and set the expiration time. Do not perform any operation and wait for the cache to expire. After the expiration time, the cache invalidation.
no problem with case one.
case two:
Set the cache and set the expiration time, then get the cache, After the expiration time, the cache cannot be invalidated.
case two is wrong.
I found that the cache expiration strategy was changed when the cache was obtained.

![image](https://user-images.githubusercontent.com/32148584/98517904-9932e900-22a9-11eb-8425-dc67d724ffd1.png)
